pom.xml

```diff
@@ -118,8 +154,17 @@
                 <plugin>
                     <groupId>org.jenkins-ci.tools</groupId>
                     <artifactId>maven-hpi-plugin</artifactId>
+                    <extensions>true</extensions>
                     <configuration>
-                        <pluginFirstClassLoader>true</pluginFirstClassLoader>
+                        <pluginFirstClassLoader>false</pluginFirstClassLoader>
```

Comment:
I tend to prefer that as it is so much simpler and easier to maintain in pom, which - given the core class leak can not be covered by JTH tests - is an advantage.